.github/workflows/mypy_primer.yml

```diff
@@ -48,10 +48,58 @@ jobs:
             --num-shards 3 --shard-index ${{ matrix.shard-index }} \
             --debug \
             --output concise \
-            | tee diff.txt
+            | tee diff_${{ matrix.shard-index }}.txt
           ) || [ $? -eq 1 ]
       - name: Upload mypy_primer diff
         uses: actions/upload-artifact@v2
         with:
-          name: mypy_primer_diff_${{ matrix.shard-index }}
-          path: diff.txt
+          name: mypy_primer_diffs
+          path: diff_${{ matrix.shard-index }}.txt
+
+  comment:
+    name: Comment
+    runs-on: ubuntu-latest
+    needs: mypy_primer
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Download diffs
+        uses: actions/download-artifact@v2
+        with:
+          name: mypy_primer_diffs
+
+      - name: Post comment
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const fs = require('fs')
+            const data = (
+              ['diff_0.txt', 'diff_1.txt']
+              .map(fileName => fs.readFileSync(fileName, { encoding: 'utf8' }))
+              .join('')
+              .substr(0, 30000)  // About 300 lines
+            )
+
+            console.log("Diff from mypy_primer:")
+            console.log(data)
+
+            let body
+            if (data.trim()) {
+              body = 'Diff from [mypy_primer](https://github.com/hauntsaninja/mypy_primer), showing the effect of this PR on open source code:\n```diff\n' + data + '```'
+            } else {
+              body = 'According to [mypy_primer](https://github.com/hauntsaninja/mypy_primer), this change has no effect on the checked open source code. 🤖🎉'
```

Comment:
nit: can we get rid of the no-op message? It can be noisy (e.g. for all the mypyc work), even with comment hiding, and no one on this repo has asked for such a thing yet.

Reply:
As it is now, this PR also introduces comment hiding to mypy. I'll delete that too, and it can be added back in a separate PR if someone wants it.

Reply:
I think that's fine to keep — or at least, I want comment hiding! :D

Reply:
I'll make a separate PR once this is done